liberty-archetype-webapp/pom.xml

```diff
@@ -9,33 +9,15 @@
     </parent>
 
     <artifactId>liberty-archetype-webapp</artifactId>
-
     <packaging>maven-archetype</packaging>
-    
     <name>Archetype - liberty-archetype-webapp</name>
-    
+
     <build>
-        <resources>
```

Comment:
What is detailing what to include in the generated project now? Does everything get included now and is that what we want?

Reply:
The resources that are supposed to be generated/copied are already specified in the [archetype-metadata.xml](https://github.com/OpenLiberty/ci.maven/blob/cff4527dd461e2ed5c675d91b5ddf64cd9a9346b/liberty-archetype-ear/src/main/resources/META-INF/maven/archetype-metadata.xml) file. Not sure how/if this was working before